### PR TITLE
fix newrelic dashboard key prop on list

### PR DESCRIPTION
**Pull request by brunocascio**
## Hey, I just made a Pull Request!

- Added `key` props to newrelic dashboard list component

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: .changeset/violet-apes-beam.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-newrelic-dashboard': patch
+---
+
+fixed required react propery "key" looping the dashboards
```
Comment (vinzscam) [inline .changeset/violet-apes-beam.md:5]:
```suggestion
Fixed React Warning: "Each child in a list should have a unique 'key' prop" during the rendering of `EntityNewRelicDashboardCard`
```

**Review by vinzscam — DISMISSED** [review]
Nice catch! Just a small suggestion about the changeset

File: .changeset/violet-apes-beam.md (hunk 1 above)
Comment (brunocascio) [reply .changeset/violet-apes-beam.md]: Thanks @vinzscam!

**Review by Rugvip — APPROVED** [review]
Nice, thank you! 👍

**Pull request merged**